citi-onboarding/Recorda-one-page
# fixed the news section

### **What I did:**
- Fixed the section responsivity

### **How to test:**
- git checkout main
- git pull
- git checkout fix/news_section
- open the client directory
- write "npm run start" at the terminal
- check if its all working
- review the code
